docs/manual/common/concepts/index.toc

```diff
@@ -1,9 +1,19 @@
-CoreConcepts:Lagom core concepts
+CoreConcepts:Introduction to Lagom concepts
 LagomDesignPhilosophy:Lagom design philosophy
-Microservices:Microservices in Lagom
 PolyglotSystems:Polyglot systems with Lagom
-Technologies:Lagom component technologies
+DevEnvOvr:Development environment overview
 BuildConcepts:Lagom build philosophy
-Immutable:Immutable Objects
-ES_CQRS:Event Sourcing and CQRS
-ServiceDiscovery: Service Discovery
+ComponentTechnologies:Component technologies
+APIOverview:API Overview
+MicroserviceSystemDesign:Designing your microservices system
+MicroserviceDesign:Sizing individual microservices
+IntAndExtComm:Internal and external communication
```

Comment:
`IntAndExtComm` and `DevEnvOvr` could use a rename (I already mentioned `DevEnvOvr` in previous comments).